modular-app/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I think this license can be removed because the whole repo is under the repo's license file